src/main/java/org/mockbukkit/mockbukkit/ServerMock.java

```diff
@@ -213,6 +213,7 @@ public class ServerMock extends Server.Spigot implements Server
 	private final @NotNull Set<OfflinePlayer> whitelistedPlayers = new LinkedHashSet<>();
 
 	private final @NotNull ServerConfiguration serverConfiguration = new ServerConfiguration();
+	private int pauseWhenEmptyTime =60 ;
```

Comment:
sorry for the preciousness
```suggestion
	private int pauseWhenEmptyTime = 60;
```

Reply:
```suggestion
	private int pauseWhenEmptyTime = 60;
```